src/map/lapack2flash/FLASH_get_set_controls.c

```diff
@@ -0,0 +1,81 @@
+/*
+
+    Copyright (C) 2023, Advanced Micro Devices, Inc.
+
+    This file is part of libflame and is available under the 3-Clause
+    BSD license, which can be found in the LICENSE file at the top-level
+    directory, or at http://opensource.org/licenses/BSD-3-Clause
+
+*/
+
+#include "FLAME.h"
+
+#ifdef FLA_ENABLE_LAPACK2FLASH // Start lapack2flash
+
+static dim_t        flash_blocksize = 1024;
+static unsigned int flash_n_threads = 1;
+static dim_t        flash_depth = 1;
+static unsigned int flash_tiles = 4;
+
+/*---------------------Get/Set Helper---------------------*/
+
+FLA_Error FLASH_set_preferred_blocksize( dim_t blocksize )
+{
+    flash_blocksize = blocksize;
+
+    return FLA_SUCCESS;
+}
+
+dim_t FLASH_get_preferred_blocksize( void )
+{
+    return flash_blocksize;
+}
+
+FLA_Error FLASH_set_n_preferred_threads( unsigned int threads )
+{
+    flash_n_threads = threads;
```

Comment:
check threads > 0 and return error if == 0

Reply:
Fixed: https://github.com/iotamudelta/libflame/pull/7/commits/cfec5dbecf82a0dbb46160d15d632f72c005f45e